### PR TITLE
KHWE-21-render-artists-from-firebase

### PR DESCRIPTION
**Pull request by WOLFRIEND**
image

### CODE REVIEW

File: src/components/Artists/Artists.html [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
         <div class="mdc-card mdc-card__primary-action mdc-ripple artists__ripple">
           <div class="mdc-card__media mdc-card__media--square artists__media">
-            <img class="artists__image" src="<%= item.cover %>" alt="artist image">
+            <img class="artists__image" src="<%= item.imageURL %>" alt="artist image">
           </div>
         </div>
       </div>
       <div class="artists__name">
-        <a class="mdc-typography mdc-typography--headline5 artists__link link" href="<%= item.link %>">
+        <a class="mdc-typography mdc-typography--headline5 artists__link link" href=#>
```
Comment (ihor4568) [inline src/components/Artists/Artists.html]: let this link point to `/artist/:artistId`

File: src/components/Artists/Artists.js [hunk 2]
```diff
@@ -81,13 +21,25 @@ export class ArtistsComponent {
     });
   }
 
-  mount() {
+  fetchArtistsData() {
+    this.state.isFetching = true;
```
Comment (ihor4568) [inline src/components/Artists/Artists.js]: please remove isFetching for now. We will deal with loader later

File: src/components/Artists/Artists.scss [hunk 3]
```diff
@@ -1,6 +1,14 @@
 .artists {
   width: 100%;
 
+  &__progressbar {
```
Comment (ihor4568) [inline src/components/Artists/Artists.scss]: why do you need these progress bar styles?

File: src/components/Artists/Artists.html (hunk 1 above)
Comment (WOLFRIEND) [reply src/components/Artists/Artists.html]: Fix. Right?
image

File: src/components/Artists/Artists.js (hunk 2 above)
Comment (WOLFRIEND) [reply src/components/Artists/Artists.js]: Fix

File: src/components/Artists/Artists.scss (hunk 3 above)
Comment (WOLFRIEND) [reply src/components/Artists/Artists.scss]: Old code. Deleted

**Review by OlegShynkarenko — APPROVED** [review]

**Pull request merged**